src/devices/Pcd8544/NokiaFont.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// The specific PCD8544 font for Nokia 5110
```

Comment:
as we talked, would it make sense for this to be provided as an external file and be shown as an example in the sample rather than having it as part of the library?

Reply:
yes and no. If this is not embedded, then there won't be any font at all in the screen, so it will not work at all to display any test at all.
You'll always has to load this font or another one to even display simple things like temperature or humidity. Which is for me extremely annoying. All LCD screen has some sort of embedded fonts always, at least minimal ones.